R/tweak-tabset.R

```diff
@@ -81,7 +81,7 @@ tablist_title <- function(tab) {
 
 # Add content of a tab to a tabset
 tablist_content <- function(tab, content, parent_id, fade) {
-  id <- section_id(tab)
+  id <-  xml2::xml_attr(tab, "id")
```

Comment:
if id is an attribute we often ask why not have a helper? today I saw https://github.com/lionel-/codegrip/blob/82d164ed91ad819587796a4053b1df5b0385c677/R/ast.R#L51, it looked nice

Reply:
IMO that would generally conceal the purpose of the code more than it reveals it. `section_id()` was only need before because it was relatively complicated to figure out the id.